tools/testbuild.sh

```diff
@@ -239,6 +244,14 @@ function build {
     fail=1
   fi
 
+  # Enable debug symbols if requested. We do this after refresh
+  # to not trigger a failure by including this feature
+
+  if [ ${FORCESYMBOLS} -eq 1 ]; then
+    kconfig-tweak --file $nuttx/.config --enable CONFIG_DEBUG_SYMBOLS
+    makefunc olddefconfig
```

Comment:
don't need olddefconfig again since kconfig-tweak will do it?
BTW, how about we replace sed with kconfig-tweak in other location?

Reply:
kconfig-tweek doesn't update the rest of the .config.  We need to run olddefconfig to enable/disable what's affected by our change.
> BTW, how about we replace sed with kconfig-tweak in other location?

I think it's a good idea to replace sed by kconfig-tweek.